packages/harmony/src/components/button/FollowButton/FollowButton.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react'
+
+import type { Meta } from '@storybook/react'
+
+import { Flex } from 'components/layout'
+
+import { FollowButton } from './FollowButton'
+import type { FollowButtonProps } from './types'
+
+const meta: Meta<typeof FollowButton> = {
+  title: 'Buttons/FollowButton',
```

Comment:
Should probably add beta here and add the BetaWarning until we get design approval and stuff

Reply:
Oh you're totally right, thank you